content/en/agent/vector_aggregation.md

```diff
@@ -52,6 +52,17 @@ logs_config:
 Where `VECTOR_HOST` is the hostname of the system running Vector and `VECTOR_PORT` is the TCP port on which
 the Vector `datadog_agent` source is listening.
 
+### Docker configuration
+
+If you are using Docker, add the following to your Agent configuration file.
+
+```
+-e DD_LOGS_CONFIG_LOGS_DD_URL=<VECTOR_HOST>:<VECTOR_PORT>
+-e DD_LOGS_CONFIG_LOGS_NO_SSL=true
+-e DD_LOGS_CONFIG_USE_HTTP=true
+-e DD_LOGS_CONFIG_USE_V2_API=false
```

Comment:
Note for later: this won't be required for later Vector release as [this PR](https://github.com/vectordotdev/vector/pull/9176) has been merged and will be included in the upcoming Vector release.

Reply:
Thanks for the heads up, I created DOCS-2832!